src/app/page.js

```diff
@@ -22,7 +23,9 @@ const Page = () => {
   return (
     <div className="flex flex-col justify-center items-center relative">
       <Navigation />
+      <ResearchNavigation />
```

Comment:
Remove this too. Don't need it on the welcome page.

Reply:
oh yea i was just testing there I forgot to remove it haha